sdk/textanalytics/Azure.AI.TextAnalytics/src/AspectSentiment.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.AI.TextAnalytics.Models;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Contains the related opinions, predicted sentiment,
+    /// confidence scores and other information about an aspect of a product.
+    /// An aspect of a product/service is a key component of that product/service.
+    /// For example in "The food at Hotel Foo is good", "food" is an aspect of
+    /// "Hotel Foo".
+    /// </summary>
+    public readonly struct AspectSentiment
+    {
+        internal AspectSentiment(TextSentiment sentiment, string text, double positiveScore, double negativeScore)
+        {
+            Sentiment = sentiment;
+            Text = text;
+            ConfidenceScores = new SentimentConfidenceScores(positiveScore, 0d, negativeScore);
+        }
+
+        internal AspectSentiment(SentenceAspect sentenceAspect)
+        {
+            Text = sentenceAspect.Text;
```

Comment:
Nit - is it worth a `sentenceAspect ?? throw new ArgumentNullException(nameof(sentenceAspect));`?  Ditto for the other models.

Reply:
Fair. It wouldn't hurt. I will add it